docs/projects/ad719x_asdz/index.rst

```diff
@@ -0,0 +1,161 @@
+.. _ad719x_asdz:
+
+AD719x-ASDZ HDL PROJECT
```

Comment:
"PROJECT" shouldn't be with capital letters, but the project name should (eg: AD719X-ASDZ). Please keep the consistency as in all of the other docs:
https://github.com/analogdevicesinc/hdl/blob/main/docs/projects/ad9081_fmca_ebz/index.rst?plain=1#L3
https://github.com/analogdevicesinc/hdl/blob/main/docs/projects/ad9783_ebz/index.rst?plain=1#L3